components/infra/docker-compose.yml

```
@@ -178,12 +178,58 @@ services:
+    command: >
+      server -s3 -dir=/data
+      -s3.port=${SEAWEEDFS_S3_PORT}
+      -s3.config=/etc/seaweedfs/s3.json
+      -master.port=${SEAWEEDFS_MASTER_PORT}
+    ports:
+      - ${SEAWEEDFS_S3_PORT}:${SEAWEEDFS_S3_PORT}
+      - ${SEAWEEDFS_MASTER_PORT}:${SEAWEEDFS_MASTER_PORT}
+    # Probe 127.0.0.1 (not localhost): SeaweedFS binds the master to IPv4 only,
+    # and busybox wget resolves localhost to [::1] first → connection refused.
+    healthcheck:
+      test: [ "CMD-SHELL", "wget --no-verbose --tries=1 --spider http://127.0.0.1:${SEAWEEDFS_MASTER_PORT}/cluster/status || exit 1" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)